pages/linux/smbclient.md

```diff
@@ -0,0 +1,19 @@
+# smbclient
+
+> FTP-like client to access SMB/CIFS resources on servers.
+
+- Connect to a share (`exit` to quit the session):
+
+`smbclient {{//server/share}}`
+
+- Connect with a username and password:
+
+`smbclient {{//server/share}} --user {{username%password}}`
```

Comment:
Is the % required ? Then it should be {{username}}%{{password}}.

Reply:
I wasn't sure how to display that... Technically `%password` is optional if you want to enter username only and use the prompt to enter the password. `{{username}}{{%password}}`?

Reply:
You can split it into 2 examples. One having {{username}}, another having {{username%password}}.